pkg/ccr/job.go

```diff
@@ -2340,21 +2340,21 @@ func (j *Job) handleAlterJob(binlog *festruct.TBinlog) error {
 		return nil
 	}
 
-	if featureSkipRollupBinlogs && alterJob.Type == record.ALTER_JOB_ROLLUP {
+	if FeatureSkipRollupBinlogs && alterJob.Type == record.ALTER_JOB_ROLLUP {
 		log.Warnf("skip rollup alter job: %s", alterJob)
 		return nil
 	}
 
 	if alterJob.Type == record.ALTER_JOB_SCHEMA_CHANGE {
-		return j.handleSchemaChange(alterJob)
+		return j.HandleSchemaChange(alterJob)
 	} else if alterJob.Type == record.ALTER_JOB_ROLLUP {
-		return j.handleAlterRollup(alterJob)
+		return j.HandleAlterRollup(alterJob)
 	} else {
 		return xerror.Errorf(xerror.Normal, "unsupported alter job type: %s", alterJob.Type)
 	}
 }
 
-func (j *Job) handleAlterRollup(alterJob *record.AlterJobV2) error {
+func (j *Job) HandleAlterRollup(alterJob *record.AlterJobV2) error {
```

Comment:
Consider moving the `HandleAlterRollup` and `HandleSchemaChange` to handle/alter_job_v2.go